products/BellHybrid/apps/application-bell-focus-timer/data/FocusTimerStyle.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <Style.hpp>
+#include "widgets/BellBaseLayout.hpp"
+
+namespace app::focusTimerStyle
+{
+    namespace runningStyle
+    {
+        namespace progress
+        {
+            constexpr inline auto radius                   = 192;
```

Comment:
To be consistent here use `inline constexpr auto` here for all variables. Also, add `U` at the end of numbers.
```suggestion
            inline constexpr auto radius                   = 192U;
```